observability/index.adoc

```diff
@@ -23,9 +23,13 @@ ifndef::openshift-dedicated,openshift-rosa[]
 * {PM-shortname-c}
 endif::openshift-dedicated,openshift-rosa[]
 
-
 {ObservabilityLongName} connects open-source observability tools and technologies to create a unified {ObservabilityShortName} solution. The components of {ObservabilityLongName} work together to help you collect, store, deliver, analyze, and visualize data.
 
+[NOTE]
+====
+With the exception of monitoring, {ObservabilityLongName} components have distinct release cycles separate from the core {product-title} release cycles. See the Red{nbsp}Hat link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] page for their release compatibility.
```

Comment:
```suggestion
Except for monitoring, {ObservabilityLongName} components have distinct release cycles separate from the core {product-title} release cycles. See the Red{nbsp}Hat link:https://access.redhat.com/support/policy/updates/openshift_operators[OpenShift Operator Life Cycles] page for their release compatibility.
```

Reply:
Thank you @Srivaralakshmi for the comment! "except for" sounded informal to me here, so I chose to use "with the exception of". In the RH style guide, I found instances of using both, so I would prefer to leave the more formal version, if that is okay with you. Thank you!